connectors/handlers/tasks.py

```diff
@@ -328,3 +328,89 @@ def slack_bot_handle_receive_message(slack_connector_id, message):
 slack_bot_handle_receive_message_prerun_notifier = publish_pre_run_task(slack_bot_handle_receive_message)
 slack_bot_handle_receive_message_failure_notifier = publish_task_failure(slack_bot_handle_receive_message)
 slack_bot_handle_receive_message_postrun_notifier = publish_post_run_task(slack_bot_handle_receive_message)
+
+
+@shared_task(max_retries=3, default_retry_delay=10)
+def pagerduty_handle_receive_message(pagerduty_connector_id, message):
+    try:
+        pagerduty_connector = get_db_connectors(connector_id=pagerduty_connector_id)
+        if not pagerduty_connector:
+            print(f"Error while handling PagerDuty handle_receive_message: Connector not found for connector_id: "
+                  f"{pagerduty_connector_id}")
+            return
+        pagerduty_connector = pagerduty_connector.first()
+        account_id = pagerduty_connector.account_id
+
+        bot_auth_token = get_db_connector_keys(account_id=account_id, connector_id=pagerduty_connector.id,
```

Comment:
Instead of calling this, do this-

pagerduty_connector_proto = pagerduty_connector.unmasked_proto
account_id =  pagerduty_connector_proto.account_id.value